pkg/converter/profileConverter.go

```diff
@@ -0,0 +1,33 @@
+package converter
+
+import (
+	"github.com/projectcalico/libcalico-go/lib/api/unversioned"
+	"github.com/projectcalico/libcalico-go/lib/api"
+	"k8s.io/client-go/pkg/api/v1"
+
+)
+const NS_POLICY_ANNOTATION = "net.beta.kubernetes.io/network-policy"
```

Comment:
I saw that in new code,  both Ingress and Egress rules are by default "allow", So the profile object will most likely be static except for name and labels, is that correct?